core/aws_ddk_core/resources/__init__.py

```diff
@@ -22,6 +22,7 @@
 from aws_ddk_core.resources._s3 import S3Factory
```

Comment:
General question on testing. I didn't write a test for this yet as we don't have AWS credentials configured in Github and am sure we won't. Is this just something we leave without a pytest as it requires using boto3 directly?

Reply:
How about a [moto](https://github.com/spulec/moto) mocked test?

Reply:
Will moto support reading lambda layers that actually exist? I might not have enough knowledge on the topic.